### DISCUSSION TITLE
Invalid callback elFinderSession::session_start_error

### DISCUSSION THREAD

**Discussion by gonandriy** [Studio-42/elFinder#3342] [Q&A] [answered]
Using with Symfony 3.4 I get PHP Warning 

`PHP Warning:  Invalid callback elFinderSession::session_start_error, cannot access protected method elFinderSession::session_start_error() in ../vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/StrictSessionHandler.php on line 101
`

Might it be worth to make the method public?

**Comment by nao-pon** [comment, marked as answer]
@gonandriy When integrating with other frameworks that use sessions, it may be convenient to use your own session handler.

- https://github.com/Studio-42/elFinder/wiki/Connector-configuration-options-2.1#session-218
